src/webhooks/pubsub/slackScores.test.ts

```diff
@@ -0,0 +1,114 @@
+import { bolt } from '@api/slack';
+import * as scoresUtils from '@utils/scores';
+
+import { triggerSlackScores } from './slackScores';
+
+describe('slackScores Tests', function () {
+  let getIssueEventsForTeamSpy, postMessageSpy;
+  beforeAll(() => {
+    getIssueEventsForTeamSpy = jest.spyOn(scoresUtils, 'getIssueEventsForTeam');
+    postMessageSpy = jest.spyOn(bolt.client.chat, 'postMessage');
+  });
+  afterEach(() => {
+    jest.clearAllMocks();
+  });
+
+  it('should handle case when no issues are returned', async () => {
+    getIssueEventsForTeamSpy.mockReturnValue([]);
+    await triggerSlackScores(null, null);
+    expect(postMessageSpy).toHaveBeenCalledWith({
+      blocks: [
+        {
+          text: {
+            emoji: true,
+            text: '🗓️ Weekly Team GitHub Engagement Scores 🗓️',
+            type: 'plain_text',
+          },
+          type: 'header',
+        },
+        {
+          text: {
+            text: `\`\`\`┌──────────────────────────────────────────────────────────────┐
+| Team                              │ GitHub Responses on Time |
+├──────────────────────────────────────────────────────────────┤
+|team-ospo                          | 0/0 (100%)               |
```

Comment:
Add space and remove redundant `team-` prefix?

```suggestion
| ospo                               | 0/0 (100%)               |
```

Reply:
done!